frontend/pages/admin/TeamManagementPage/TeamManagementPage.tsx

```diff
@@ -253,7 +253,7 @@ const TeamManagementPage = (): JSX.Element => {
           showMarkAllPages={false}
           isAllPagesSelected={false}
           searchable={teams && teams.length > 0 && searchString !== ""}
-          disablePagination
```

Comment:
Strange, I wonder why this was disabled initially.

Reply:
IIRC, stylistically, Figma was missing the pagination, which kinda made sense when we had 100 teams on a page